06_modules_lab/02_bonus.py

```diff
@@ -1,10 +1,12 @@
-""" Write a program that reads 2 numbers from sys.argv
-and prints their sum.
-Bonus: Print error messages for invalid inputs.
-
-To print error messages we'll have to use a concept not yet learned in the
-course, and which will only be presented later: Exceptions.
-We'll tap into python's error handling and change its default
-error message to something more meaningful.
-"""
-
+import sys
+

```

Comment:
Now that you found try/except, you can use them to write cleaner code:
Remove the `if len(sys.argv)...` and wrap everything in a try/except block.
If argument count or type is invalid, you'll end up in the except and can handle it only once (instead of your current code which duplicates the printing of error message)